.github/workflows/static_analysis.yml

```diff
@@ -0,0 +1,31 @@
+name: Static Analysis
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  coverage:
+    name: SonarQube - (master)
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # For sonar-scanner to work properly we can't use a shallow fetch.
+          fetch-depth: 0
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm ci
+      - name: Test coverage
+        run: npm run coverage:ci
```

Comment:
So in JS/TS case it doesn't require to generate an output file for Sonar cloud to pick up?